static/js/stream_edit.js

```diff
@@ -156,7 +156,7 @@ function show_subscription_settings(sub_row) {
     var color = stream_data.get_color(sub.name);
     stream_color.set_colorpicker_color(colorpicker, color);
 
-    if (!sub.render_subscribers) {
+    if (!sub.render_subscribers || !sub.can_add_subscribers) {
```

Comment:
We should return from this function if user can't access subscribers, to prevent "DOM element not found" error. Cause we only add subscribers-table DOM element to setting, if user can access them.